transcendence-app/src/game/game.gateway.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  ClassSerializerInterceptor,
+  ParseUUIDPipe,
+  UseFilters,
+  UseGuards,
+  UseInterceptors,
+} from '@nestjs/common';
+import {
+  ConnectedSocket,
+  MessageBody,
+  SubscribeMessage,
+  WebSocketGateway,
+  WebSocketServer,
+  // WsException,
+} from '@nestjs/websockets';
+import { Socket, Server } from 'socket.io';
+import { BadRequestTransformationFilter } from '../shared/filters/bad-request-transformation.filter';
+import { WsAuthenticatedGuard } from '../shared/guards/ws-authenticated.guard';
+import { GameInputDto } from './dto/game-input.dto';
+
+@WebSocketGateway({ path: '/api/v1/socket.io' })
+@UseGuards(WsAuthenticatedGuard)
+@UseInterceptors(ClassSerializerInterceptor)
+@UseFilters(BadRequestTransformationFilter)
+export class GameGateway {
```

Comment:
Don't pay much attention to this new Game module, is just a dummy scaffolding I made to start thinking how is going to be the  ws connection between client and server